### ISSUE TITLE
Images not loading when outside local network

### ISSUE THREAD

**Issue by GoldieLocks89** [0x2142/frigate-notify#142]
Images are not loading in the gotify message whenever I'm outside my local network. I can click on both the "Camera" and "Clip" links and they open fine in the browser. But images won't show in gotify.
Inside my local network they open properly.
Potentially this is a URL set wrongly somewhere, which would be weird as the clip Url is correct, but how can I debug this? Is there any way of seeing/logging the raw content of the message that's sent so I can debug what's wrong?

**Comment by GoldieLocks89** [comment]
> Since you said you can load the clip & camera links, I assume you have some method of accessing Frigate 
> 

Yes, a reverse proxy secured with a client certificate. 

> The link sent to Gotify uses the `frigate > server` config setting, and not the `public_url` if you're using that (though maybe I should update Gotify to use that one instead).
> 

Yeah that'd be awesome.

> Also I was working yesterday on some updated logging that will contain the raw HTTP requests sent & received, but I'm not quite done with it yet. Should have that done soon & available to use in the next release - which I'm hoping to get out in the next few days.
>

Thay would be helpful too, thank you :)

**Comment by 0x2142** [comment]
Thanks for the feedback. This has been fixed on the latest `dev` branch images if you need it sooner. Otherwise, I should be able to cut a new release in the next few days after some additional testing.

**Comment by bobemoe** [comment]
I had similar issue with Gotify. 

I considered making a custom template to set the public_url  but in my testing sending a Gotify notification containing a snapshot link (e.g. http://frigate:8971/api/events/1728292621.263665-wxrp0p/snapshot.jpg) it still did not give particularly reliable access to the snapshot, because my Frigare is password protected and you just get a 401 and not even redirected to the login page. Maybe ok if your browser is always logged in.

I did consider trying to pass the user:password in on the URL, or hacking the reverse proxy config to make /api/events/*/snapshot.jpg not require auth, which I think might work, but..

After fining this https://github.com/0x2142/frigate-notify/issues/101 I ended up switching to ntfy and I couldn't be happier! Instant snapshots embedded in the notification right on my mobile home screen :)
